library/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -1085,6 +1096,9 @@ private void drawEventTitle(WeekViewEvent event, RectF rect, Canvas canvas, floa
         int availableWidth = (int) (rect.right - originalLeft - mEventPadding * 2);
 
         // Get text dimensions.
+        if(textColorPicker != null) {
```

Comment:
This if should be placed at line 554 or something, now it's set every time a title is drawn